### ISSUE TITLE
Mention in your documentation TumblThree - a Windows app alternative

### ISSUE THREAD

**Issue by edadams** [bbolli/tumblr-utils#156]
This repo is no doubt seeing a lot of activity now that there is something of a mass exodus away from Tumblr. 

I noticed that the specific page I got linked to didn't have a mention of [TumblThree](https://www.jzab.de/content/tumblthree), which basically does an equivalent of saving all of a Tumblr user's content (including reblogs, etc.) locally. The advantage is that it has a GUI, doesn't require installing or using Python, etc. It has a pretty advanced feature set and can even download external resources.

Your resources are very helpful for people who want to export their content especially considering how ineffective Tumblr's built-in export tools are, but I think that in the interests of being as helpful as possible it is definitely worth mentioning TumblThree (I can personally vouch for its effectiveness as it created a comprehensive backup of my blog for me). 

Thanks for everything you've done.

**Comment by Cebtenzzre** [comment]
The disadvantages are that TumblThree doesn't save likes in the same way (it scrapes images and videos but doesn't get the actual posts/text), and that it tends to lag a lot (at least in my Windows 7 VM).  Though it scrapes likes much faster due to not being hindered by API throttling, can be used to scrape hidden blogs, and is better at remembering where it left off.

**Comment by edadams** [comment]
Fair enough. I still think it’s worth mentioning as a more user friendly
alternative (anything is better than nothing) for people who need to get
off Tumblr quickly.

Perhaps the limitation that it can’t work with likes should also be
mentioned in this case? :)

**Comment by waldens** [comment]
@edadams TumblThree is able to download what u liked, just read section about liked-by in the README. Still, both tools have some trouble(or bugs? dont know) when backup our tumblr account now: #151 

Other options:
• [easychen/tumblr-like-exporter: Download tumblr photos you liked , or others liked . Or even images in your blog . It supports videos now , see README to enable it](https://github.com/easychen/tumblr-like-exporter)
• [dixudx/tumblr-crawler: Easily download all the photos/videos from tumblr blogs.](https://github.com/dixudx/tumblr-crawler)
• [cyang812/get_tumblr_likes](https://github.com/cyang812/get_tumblr_likes)

**Comment by shelbylee-final** [comment]
imo the biggest advantage of TumblrThree in this mass exodus and big time crunch is that it retrieves read-more content - tumblr-utils just redirects and a lot those redirects will be lost when posts start getting removed either by users or tumblr staff.

**Comment by Cebtenzzre** [comment]
@shelbylee-final What do you mean by "read-more content"?  My first interpretation was "content located after a read-more/keep-reading button".  But I tried downloading a post with a read-more button using tumblr-utils, and all of the images and text after it were saved properly.  There was no read-more button in the saved HTML, let alone any kind of redirect.

If somehow you're referring to TumblThree's ability to download images and videos from external sources, tumblr-utils can do that too.  It is however less selective; there are no site-specific (Imgur, Gfycat, etc.) options in tumblr-utils.

**Comment by shelbylee-final** [comment]
Let me clarfy. TumblrThree does read mores for external blogs when you log in. Which is a huge benifit. 

E.g. http://gaiasanta.tumblr.com/post/154928637994/merry-christmas-welcome-to-shinra

Tumblr-utils only backs up the gaiasanta blog post and links out to the read more blog post (this example does not like to hidden blog). But ideally it should download the original posters read more, the the reblogged post. 

(There's already an Issue logged for Read Mores that doesn't have any traction or commits)

**Comment by shelbylee-final** [comment]
I just realized that's not a readmore, just a link embedded with a read more link. Sorry for the confusion!
